examples/refreshAccessToken.js

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const OAuth2Client = require('../build/src/index').OAuth2Client;
+const http = require('http');
+const url = require('url');
+const querystring = require('querystring');
+const opn = require('opn');
+
+// Download your OAuth2 configuration from the Google
+const keys = require('./oauth2.keys.json');
+
+/**
+ * Start by acquiring a pre-authenticated oAuth2 client.
+ */
+async function main() {
+  try {
+    const oAuth2Client = await getAuthenticatedClient();
+    // If you're going to save the refresh_token, make sure
+    // to put it somewhere safe!
+    console.log(`Refresh Token: ${oAuth2Client.credentials.refresh_token}`);
+    console.log(`Expiration: ${oAuth2Client.credentials.expiry_date}`);
+    console.log('Refreshing access token ...');
+    const res = await oAuth2Client.refreshAccessToken();
```

Comment:
@nolanmar511 perhaps this pattern can be temporary workaround while #208 is figured out?

Reply:
Possibly, but it would require this in @google-cloud/common, which would require exposing this in google-auto-auth. 
Depending on the time to resolve #208, I'm not certain that exposing this would be worth the effort.